scripts/clap

```diff
@@ -211,9 +221,19 @@ class Git(object):
 
 
 class Repos(object):
-    def __init__(self, branch=MASTER, dev=True, requirements=None):
+    def __init__(self,
+                 branch=MASTER,
+                 dev=True,
+                 requirements=None,
+                 plugins=True):
```

Comment:
I will default plugins to False, because currently must places will not need this

Reply:
But also this is a required param, so you can leave it with no default